src/test/kotlin/racingcar/domain/AnalyzerTest.kt

```diff
@@ -0,0 +1,173 @@
+package racingcar.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+
+import org.junit.jupiter.api.Test
+import racingcar.data.Car
+
+class AnalyzerTest {
+
+    private lateinit var analyzer: Analyzer
+
+    @BeforeEach
+    fun setUp() {
+        analyzer = Analyzer()
+    }
+
```

Comment:
18-68 Line을 ParameterizedTest와 ValueSource를 써서 하나로 만들 수 있습니다. 

```suggestion
    @ParameterizedTest
    @ValueSource(strings = ["0, false", "3, false", "4, true", "9, true"])
    @DisplayName("Analyzer : isMoveAllowed()")
    fun `자동차가 움직일 수 있는지 확인하는 함수에 value를 입력하면 false를 반환한다`(pair: String) {
        // given
        val (pickedNum, expected) = pair.split(", ")

        // when
        val actual = analyzer.isMoveAllowed(pickedNum.toInt())

        // then
        assertThat(actual).isEqualTo(expected.toBoolean())
    }
```

또한, 이렇게 작성할 경우 아래 이미지와 같이 테스트 결과를 직관적으로 확인할 수 있어서 오류 대처에 용이합니다!
![image](https://github.com/woowacourse-precourse/kotlin-racingcar-6/assets/52482206/1aa32635-4684-473d-9239-b96202c03ec7)

Reply:
Gradel - run test using 값을 Intellij IDEA로 설정해야 DisplayName과 ValueSource 값들이 보이네요.
원래 안 보이는 줄 알았는데, 마지막에 테스트 코드 오류로 찾아보다가 알았어요🤣

![gradle](https://github.com/woowacourse-precourse/kotlin-racingcar-6/assets/48354989/8cb7d5a3-61bc-4a79-b573-ee7c2f133b7b)